Automattic/jetpack
# Product management controls: unit tests for memberships products store

Added unit test to some of our product store components.

#### Changes proposed in this Pull Request:

* Unit test for product store

#### Jetpack product discussion



#### Does this pull request change what data or activity we track or use?



#### Testing instructions:






* Checkout the repo
* Run tests
* All tests should pass
